openmetadata-ui/src/main/resources/ui/playwright/utils/user.ts

```diff
@@ -575,6 +575,7 @@ export const checkStewardServicesPermissions = async (page: Page) => {
   // Click on the sidebar item for Explore again
   await sidebarClick(page, SidebarItem.EXPLORE);
 
+  await page.waitForResponse('/api/v1/search/query?q=*');
```

Comment:
```suggestion
 const queryResponse =  page.waitForResponse('/api/v1/search/query?q=*');
 await sidebarClick(page, SidebarItem.EXPLORE);
 await queryResponse;
 
```

Reply:
@shrushti2000 let's await like this, as in the previous case, there can be a chance as, before reaching API await line the API might be resolved, and test will broke on the await line.

Reply:
Agree with @Ashish8689, Please update the changes @shrushti2000